docs/par2cron_completion_bash.md

```
@@ -0,0 +1,50 @@
+#### Linux:
+
+	par2cron completion bash > /etc/bash_completion.d/par2cron
+
+#### macOS:
+
+	par2cron completion bash > $(brew --prefix)/etc/bash_completion.d/par2cron
+
+You will need to start a new shell for this setup to take effect.
+
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)